### DISCUSSION TITLE
zh/guide/feature/comment

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#423] [Announcements]
# zh/guide/feature/comment

通过内置 ，vuepress-theme-hope 实现了评论功能。 相关信息 vuepress-theme-hope 将主题选项中的 plugins.comment 作为插件选项提供给 vuepress-plugin-comment2。

https://theme-hope.vuejs.press/zh/guide/feature/comment.html

**Comment by liub1934** [comment]
提个优化的建议，能不能在返回顶部的按钮类似的位置加一个按钮，点击可以直达评论
